django/db/models/expressions.py

```diff
@@ -1107,6 +1123,13 @@ def as_sql(self, compiler, connection):
         return "*", []
 
 
+class DB_DEFAULT(Expression):
+    """Placeholder expression for the database default in an insert query."""
+
+    def as_sql(self, compiler, connection):
+        return "DEFAULT", []
```

Comment:
Have you seen #13580? Do you think we could add the `Default()` expression in advance and use it in both cases? :thinking:

Reply:
I had not seen that - I'll think about it.

Reply:
I think we could, but I don't think there's much value in that. If that PR lands first I can easily refactor this one to use `Default` and vice-versa.